deps/build.jl

```diff
@@ -9,7 +9,7 @@ else
 
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/windows/amd64/", os = :Windows)
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/macosx/x86_64/", os = :Darwin)
-    @linux_only push!(BinDeps.defaults, Binaries)
+    @compat @static is_linux() ? push!(BinDeps.defaults, Binaries) : nothing
```

Comment:
also https://github.com/JuliaLang/Compat.jl/pull/209 for `is_linux` which was 0.7.20